Source/DafnyDriver/DafnyDriver.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Dafny\DafnyPipeline.csproj" />
     <ProjectReference Include="..\DafnyTestGeneration\DafnyTestGeneration.csproj" />
+    <ProjectReference Include="..\DafnyServer\DafnyServer.csproj" />
```

Comment:
For the record I'm fine with this for now, but would really like to more the counter-example extraction code somewhere more generic in the future, either in the language server or in the core `DafnyPipeline` project itself.